### PR TITLE
Add test for checkIfEmptyRow method in StringHelper.java #8279

### LINKED ISSUE #8279: Add test for checkIfEmptyRow method in StringHelper.java
**Description of feature/enhancement**

There is currently no test for `checkIfEmptyRow` in StringHelper.java, and the tests pass even with a wrong implementation.

**Justification**

Catches bugs when refactoring that section, as well as reviewer lapses (demonstrated in #8258).

### PR DESCRIPTION
**Pull request by pierreSquared**
Fixes #8279

Added test cases to the testCsvToHtmlTable method that cover the private checkIfEmptyRow method.
These test cases check whether checkIfEmptyRow method returns true given row with no data, or with only empty table data cells and false when there is at least one non-empty data cell.

### CODE REVIEW

**Comment by pierreSquared** [comment]
Ready for review.

**Comment by xpdavid** [comment]
Since the inputs are all valid, can you add your test data to the existing one ?

```java
    @Test
    public void testCsvToHtmlTable() {
        String csvText = "ColHeader1, ColHeader2, ColHeader3, ColHeader4" + Const.EOL
                         + "\"Data 1-1\", \"Data 1\"\"2\", \"Data 1,3\", \"Data 1\"\"\"\"4\"" + Const.EOL
                         + "Data 2-1, Data 2-2, Data 2-3, \"Data 2-4\"\"\"" + Const.EOL
                         + "Data 3-1, Data 3-2, Data 3-3, Data 3-4" + Const.EOL;
                         + your newly added test data rows
        String htmlText = StringHelper.csvToHtmlTable(csvText);
        String expectedHtmlText = "<table class=\"table table-bordered table-striped table-condensed\">"
                                      + "<tr>"
                                          + "<td>ColHeader1</td>"
                                          + "<td> ColHeader2</td>"
                                          + "<td> ColHeader3</td>"
                                          + "<td>ColHeader4</td>"
                                      + "</tr>"
                                      + "<tr>"
                                          + "<td>Data 1-1</td>"
                                          + "<td> Data 1&quot;2</td>"
                                          + "<td> Data 1,3</td>"
                                          + "<td>Data 1&quot;&quot;4</td>"
                                      + "</tr>"
                                      + "<tr>"
                                          + "<td>Data 2-1</td>"
                                          + "<td> Data 2-2</td>"
                                          + "<td> Data 2-3</td>"
                                          + "<td>Data 2-4&quot;</td>"
                                      + "</tr>"
                                      + "<tr>"
                                          + "<td>Data 3-1</td>"
                                          + "<td> Data 3-2</td>"
                                          + "<td> Data 3-3</td>"
                                          + "<td>Data 3-4</td>"
                                      + "</tr>"
                                      + ..... more expected tr
                                  + "</table>";
        assertEquals(expectedHtmlText, htmlText);
    }
```

File: src/test/java/teammates/test/cases/util/StringHelperTest.java [hunk 1]
```diff
@@ -404,6 +404,65 @@ public void testCsvToHtmlTable() {
                                       + "</tr>"
                                   + "</table>";
         assertEquals(expectedHtmlText, htmlText);
+
+        csvText = "ColHeader1, ColHeader2, ColHeader3" + Const.EOL
```
Comment (xpdavid) [inline src/test/java/teammates/test/cases/util/StringHelperTest.java]: Is this `csvText` a valid csv format?

**Comment by pierreSquared** [comment]
Added the test data to existing test case and removed an invalid test. Ready for review.

**Pull request merged**